ompy/extractor.py

```diff
@@ -96,25 +104,36 @@ def __init__(self, ensemble: Optional[Ensemble] = None,
         self.extend_fit_by_resolution: bool = False
         self.resolution_Ex = 150  # keV
 
+        self.rel_err_missing = 0.3
+
     def __call__(self, ensemble: Optional[Ensemble] = None,
                  trapezoid: Optional[Action] = None):
         return self.extract_from(ensemble, trapezoid)
 
     def extract_from(self, ensemble: Optional[Ensemble] = None,
                      trapezoid: Optional[Action] = None,
+                     error_estimator: Optional[ErrorFinder] = None,
                      regenerate: Optional[bool] = None):
         """Decompose each first generation matrix in an Ensemble
 
         If `regenerate` is `True` it saves the extracted nld and gsf to file,
         or loads them if already generated. Exposes the vectors in the
         attributes self.nld and self.gsf.
 
+        If the error_estimator attribute or argument is set then relative
+        errors will be estimated using the provided algorithm. Points in the
+        nld and gsf that the algorithm are unable to estimate will be set to
```

Comment:
Suggestion: /Relative error/ of points (...) will be set to `self.rel_err_missing`.